### DISCUSSION TITLE
Browser Notifications?

### DISCUSSION THREAD

**Discussion by tornupnegatives** [balzack/databag#81] [Ideas]
How difficult would it be to serve notifications from the browser? I imagine that an open Databag tab could request permission to show notifications, or at least play a sound when a new message arrives.

**Comment by balzack** [comment]
Integration with firefox and chrome is pretty easy to do, safari is a bit more involved. That feature is currently in my backlog about 2 releases away (= ~4 months). If that feature seems important, I'll bump it forward.

**Reply by TheHung184** [reply]
Hi, i also interest in this feature, if it is possible i think you can release for Firefox and Chrome first. Looking forward to it. ^  ^

**Comment by balzack** [comment]
I just published a new version with this feature. Depending on the container management platform you use, it should be available in the next few days.

Note: for existing logged in sessions, you will need to logout then back in. You can enable the notifications under the profile/account settings.

**Reply by TheHung184** [reply]
Hi, I have update to latest and see the notification. Thank you for your effort :D

**Reply by balzack** [reply]
Thank you for your help with the project!
